### DISCUSSION TITLE
Unresolved conversations do not prevent pull request merging

### DISCUSSION THREAD

**Discussion by SamuelCabralCruz** [community/community#7827] [Pull Requests]
Hi, 

I just tried the repository settings to prevent the merge of pull requests with unresolved conversations.
Even if, I see that unresolved conversations are being properly detected, I don't see any way to prevent the merge of the pull request.

Best regards,
Samuel

**Comment by eperret** [comment]
I am still seeing this issue.  Did git provide a solution or workaround?

**Comment by rjhuijsman** [comment]
Our company uses [MergeQueue](https://docs.aviator.co/getting-started/mergequeue) to work around this. We use [the `publish_status_check` feature](https://docs.aviator.co/reference/merge-rules), require the resulting check to pass in GitHub's branch protection rules, and have MergeQueue enforce that [all conversations are resolved with `conversation_resolution_required`](https://docs.aviator.co/reference/merge-rules#preconditions) before its check goes green.

That said, we were already using MergeQueue for other features before we also used it to work around this issue. YMMV!
